src/components/TxLyraSign.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const signWithLyra = async (xdr, setTxResult) => {
+      let res = {transactionStatus: "" };
+
+    try {
+      res = await window.lyra.submitTransaction({
```

Comment:
Marking this as a WIP API, this is proof of concept atm. We should probably talk through possible use cases apps have, naming.